resources/did-configuration/index.html

```diff
@@ -648,7 +648,8 @@ <h2>
           <li>The object MUST contain an `id` property, and its value MUST be a valid DID URL reference.</li>
           <li>The object MUST contain a `type` property, and its value MUST be the string `"LinkedDomains"`.</li>
           <li>The object MUST contain an `origins` property, and its value MUST be an array of one or more 
-            <a href="https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy">domain origins</a>.</li>
+            <a href="https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy">domain origins</a>,
+            or a single `endpoint` property conveying a domain prefixed by `https://`.</li>
```

Comment:
```suggestion
            or a single `serviceEndpoint` property conveying a domain prefixed by `https://`.</li>
```

Reply:
@jmandel is this what you meant, "endpoint" / "origin" are not defined, `serviceEndpoint` is... https://w3c.github.io/did-spec-registries/#serviceendpoint

Reply:
Yes, precisely -- thanks for fixing my careless slip!